code/datums/traits/positive_quirk.dm

```diff
@@ -123,10 +124,20 @@
 	languages_possible = languages_possible - H.language_holder.spoken_languages
 	languages_possible = languages_possible - H.language_holder.blocked_languages
 	if(length(languages_possible))
-		var/datum/language/random_language = pick(languages_possible)
-		H.grant_language(random_language, TRUE, TRUE, LANGUAGE_MULTILINGUAL)
+		known_language = pick(languages_possible)
+		quirk_target.grant_language(known_language, TRUE, TRUE, LANGUAGE_MULTILINGUAL)
 //Credit To Yowii/Yoworii/Yorii for a much more streamlined method of language library building
 
+/datum/quirk/multilingual/add()
```

Comment:
I think this can possibly make a monkified person talk this language.

Reply:
you are a wizard for seeing this